BindFamilyLockBonusActivationActivity/index.ts

```diff
@@ -0,0 +1,57 @@
+import { AzureFunction, Context } from "@azure/functions";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import { BonusActivation } from "../generated/models/BonusActivation";
+import { BonusActivationStatusEnum } from "../generated/models/BonusActivationStatus";
+import { CosmosDbDocumentCollection, toBaseDoc } from "../services/cosmosdb";
+import { generateFamilyUID } from "../utils/hash";
+
+const index: AzureFunction = async (_: Context, input: unknown) => {
+  const decoded = CosmosDbDocumentCollection.decode(input);
+  if (decoded.isLeft()) {
+    throw Error(
+      `BindFamilyLockBonusActivationActivity: cannot decode input [${readableReport(
+        decoded.value
+      )}]`
+    );
+  }
+  const documents = decoded.value.reduce(
+    (prev, rawDocument) => {
+      const errorOrBonusActivation = BonusActivation.decode(rawDocument);
+      // Skip invalid documents and BonusActivation with state not PROCESSING
```

Comment:
why do we skip status == Processing?

Reply:
Fixed into 28809ac

Reply:
Now all the status changes are tracked, with multiple records for the same `PartitionKey`

Reply:
yes we want to keep only the latest one

Reply:
fixed in 1c124ec